build.gradle

```diff
@@ -290,6 +293,47 @@ subprojects {
                     from "$rootDir/NOTICE"
                 }
             }
+
+            // new projects
+            if (["micrometer-commons", "micrometer-binders"].every { it != project.name }) {
+                apply plugin: 'me.champeau.gradle.japicmp'
+                apply plugin: 'de.undercouch.download'
+
+                task downloadBaseline(type: Download) {
+                    onlyIf {
+                        if (project.gradle.startParameter.isOffline()) {
+                            println "Offline: skipping downloading of baseline and JAPICMP"
+                            return false
+                        } else if ("$compatibleVersion" == "SKIP") {
+                            println "SKIP: Instructed to skip the baseline comparison"
+                            return false
+                        } else {
+                            println "Will download and perform baseline comparison with ${compatibleVersion}"
+                            return true
+                        }
+                    }
+
+                    onlyIfNewer true
+                    compress true
+                    src "${repositories.mavenCentral().url}io/micrometer/${project.name}/${compatibleVersion}/${project.name}-${compatibleVersion}.jar"
```

Comment:
Does `japicmp` has a `configuration`?
If so we can add a dependency like `japicmp 'io.micrometer:micrometer-core:1.8.4'` so gradle will download the jar for us and we don't need the download plugin.

Reply:
I've copied it over from reactor-netty